packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py

```diff
@@ -53,7 +53,7 @@ async def _call(
             user_id=user_id,
             limit=limit,
             offset=offset,
-            timeout_s=2 * RPC_REQUEST_DEFAULT_TIMEOUT_S,
+            timeout_s=4 * RPC_REQUEST_DEFAULT_TIMEOUT_S,
```

Comment:
minor: just define this as a new constant above in this file and explicitly put the value in.

Reply:
This is still experimental, we will eventually try to bring it down to one